imgp.1

```diff
@@ -56,7 +56,7 @@ Output resolution in HRESxVRES or percentage.
 Rotate clockwise by a specified angle (in degrees). Negative inputs rotate anti-clockwise. Rotation by 0 degree is not allowed.
 .TP
 .BI "-q, --quality=" N
-Save the image with a specified quality factor N. Only JPEG image support for this feature is present in PIL currently. Default value is 75.
+Save the image with a specified quality factor N. JPEG only. Default value is 75.
```

Comment:
Please mention the range. Maybe:

`Save the image with a specified quality factor N (scale: 1-95, default 75). JPEG only.`